misc/snapshotter/Dockerfile

```diff
@@ -1,31 +1,43 @@
 FROM alpine:3.17.0 AS sourcer
 
 ARG NYDUS_VER=v2.1.5
-
 RUN apk add --no-cache curl
 RUN apk add --no-cache --upgrade grep
-RUN wget https://github.com/dragonflyoss/nydus/releases/download/$NYDUS_VER/nydus-static-$NYDUS_VER-linux-amd64.tgz && \
+RUN curl -OL https://github.com/dragonflyoss/nydus/releases/download/$NYDUS_VER/nydus-static-$NYDUS_VER-linux-amd64.tgz && \
     echo $NYDUS_VER > /.nydus_version && \
     tar xzf nydus-static-$NYDUS_VER-linux-amd64.tgz && \
-    rm nydus-static-$NYDUS_VER-linux-amd64.tgz
-RUN mv nydus-static/* /
+    rm nydus-static-$NYDUS_VER-linux-amd64.tgz && \
+    mv nydus-static/* / && \
+    curl -OL https://github.com/liubin/toml-cli/releases/download/v0.0.7/toml-cli-v0.0.7-linux-amd64.tgz && \
+    tar xzf toml-cli-v0.0.7-linux-amd64.tgz && \
+    mv toml-cli/toml / && \
+    rm toml-cli-v0.0.7-linux-amd64.tgz
 
 FROM alpine:3.17.0
 
 WORKDIR /root/
-
-RUN apk add --no-cache libc6-compat
+RUN apk add --no-cache libc6-compat bash jq gcompat
 
 VOLUME /var/lib/containerd-nydus /run/containerd-nydus
 
 COPY --from=sourcer /.nydus_version /.nydus_version
 
-RUN mkdir -p /usr/local/bin/ /etc/nydus/ /var/lib/containerd-nydus/cache /tmp/blobs/
-COPY --from=sourcer /nydus* /usr/local/bin/
-COPY containerd-nydus-grpc /usr/local/bin/
-RUN chmod +x /usr/local/bin/containerd-nydus-grpc
+RUN mkdir -p /usr/bin/ /opt/nydus/ /var/lib/containerd-nydus/cache /tmp/blobs/
+COPY --from=sourcer /nydus* /usr/bin/
+COPY --from=sourcer /toml /usr/bin/
+COPY containerd-nydus-grpc /usr/bin/
```

Comment:
I'm not familiar with this repository, so not sure if I'm missing something (probably I am).... but I was not able to test the entire solution because I can't build the image:

```
"/containerd-nydus-grpc": not found
```

Do you mind to share what I'm missing?

Reply:
I guess you should build this project binaries and copy `containerd-nydus-grpc` over `misc/snaphotter`. Thanks for giving it a try @beraldoleal !

Reply:
@wainersm I tried that, but the binary was still missing. @ChengyuZhu6 do you mind sharing if we need any extra step to make this passing? Or maybe a README.md here with a quick/basic instructions?

Reply:
Thanks for your informations. The following are steps to build the snapshotter docker image: 
$ make #build snapshotter binaries
$ copy bin/* misc/snapshotter/
$ pushd misc/snapshotter/
$ docker build -t ghcr.io/containerd/nydus-snapshotter:latest .
$ popd

Reply:
It would be nice then to add those instructions to a README on `misc/snapshotter`, not fancy, just this. Alternatively, add those steps as part of a `make snapshotter` target.

Reply:
> ```shell
> $ make  # build snapshotter binaries
> $ cp bin/* misc/snapshotter/
> $ pushd misc/snapshotter/
> $ docker build -t ghcr.io/containerd/nydus-snapshotter:latest .
> $ popd
> ```

Done.

Reply:
> It would be nice then to add those instructions to a README on `misc/snapshotter`, not fancy, just this. Alternatively, add those steps as part of a `make snapshotter` target.

Done.